src/index.ts

```diff
@@ -202,7 +202,11 @@ class ClientApp {
             if ('gcHostOriginQueryParam' in cfg || 'gcTargetEnvQueryParam' in cfg) {
                 this.assertNonEmptyString(cfg.gcHostOriginQueryParam, 'host origin query param name');
```

Comment:
@carolinepar ... these are now duplicated below on 207 and 208.  That's good as they'll typeguard the variables before being passed to the lookup function.  But, I think we can remove these now, right?

Reply:
nvmd ... just realized this is asserting the param name in the config.  not the value.  nothing to see here :face_palm: